src/app/routes/utils/fetchPageData/utils/getRouteProps/routeFallbackParams/index.js

```diff
@@ -1,8 +1,10 @@
+import Url from 'url-parse';
 import services from '#lib/config/services/loadableConfig';
 
 export const fallbackAmpParam = url => {
+  const { pathname } = new Url(url, true);
   const ampRegex = /\.amp$/;
-  return ampRegex.test(url);
+  return ampRegex.test(pathname);
```

Comment:
I think we might need to update the regex to cater for `/amp` as well, because the csp issue is still apparent for `http://localhost:7080/arabic/world-23278971/amp?renderer_env=test` which usually resolves to amp

Reply:
I didn't even know we supported `/amp`, can you raise an issue for this @rhenshaw56 ?

Reply:
`/amp` at the end of the url is new to me. Our legacy AMP stack had `/amp` right after the service name

Reply:
Just confirmed on test (https://www.test.bbc.com/gahuza/48118437/amp), we are not supporting `/amp`, not entirely sure how that resolves to amp locally.

Reply:
Mind creating a bug ticket so that can be looked into another time @rhenshaw56 ?